### DISCUSSION TITLE
Single membrane 10-hole flute mixed with 6-hole fingerings

### DISCUSSION THREAD

**Discussion by chihowkwan** [edwardkort/WWIDesigner#160] [Show and tell]
Single membrane 10-hole flute mixed with 6-hole fingerings

Hello, I would like to share my recent work using WIDesigner as the major design tool with you.

![tempImagevn373b](https://github.com/edwardkort/WWIDesigner/assets/85783979/28741d75-2037-4b6a-b739-2218a6473f21)

I recorded two versions of show and tell videos. This is the English version:

https://youtu.be/DOCa5Ehmx7g

And this is the Chinese version, showing the other flute:

https://youtu.be/3jmWk7gvnh0

The design of this flute model is in fact very complex and is mainly done by my friend Henry who did not appear in these videos. He often said it caused headache to him. My role is making and this might not be easy as well.

Maybe you are interested in the design. Based on the relatively simple model of toneholes in WIDesigner, Henry made them oval for better feeling of finger covering, tilted for the limited length of finger extension, undercut for limited sizes of fingers, and raised for a sum of those concerns. He has a bunch of description for each fingerhole on how to manufacture each. I believe he has his ways of transforming those effects from numbers used in WIDesigner to application.

Because WIDesigner does not have the option to add a Chinese flute dizi membrane and this membrane, we learned, causes tuning issues, Henry made a design which is somewhat out of tune on WIDeisnger’s tuning pattern. It was meant to compensate for the effect of the membrane. We thought the result was great though it caused further headache to him, he said.

The design data we created through WIDesigner is the core of these flutes. We are grateful that we can use it. Thank you. We will continue to use it and hope our flutes meet players’ need!

**Comment by edwardkort** [comment]
Allen,

That is a very creative design. I especially like the fingering pattern: 6 holes yield a minor flute, the additional 4 holes fill in the chromatic notes. You might consider providing plugs for those 4 holes so that beginners have an easier time. This is a common practice in the Native American Style Flute community.

If you consider the membrane hole as an eleventh hole, but always closed in the fingering patterns, does this help with the tuning accuracy predicted by WIDesigner?

Nice work!

Cheers,
Edward Kort

**Comment by bhp1** [comment]
Those are nice looking flutes.

WIDesigner does go only so far in modelling the holes you are dealing with.  While it can model holes of different depths, it doesn't cover oval holes, angled holes, or undercutting... or the hole that the dizi membrane covers.  As Edward suggests, you might get a bit closer by modelling the dizi hole as an always-closed hole, but I suspect that the flexible membrane will make it behave somewhere between an open and a closed hole.  An angled hole could be modeled as a straight hole at the position of the inside end of the angled hole with a hole height equal to the length of the angled hole.

Best of luck with your enterprise.

**Comment by chihowkwan** [comment]
Thank you for your praise! The idea of mixing 6-hole fingerings to a 10-hole flute was thought by our Chinese flute player friends. Developing this idea is truly difficult and we will continue to think about possible improvements!

Mr. Kort’s advice on plugs made me think more about this design and found out that I said “6 of them are major fingerholes” is probably wrong. I was in a hurry when I wrote the script and forgot that the hole for the right hand ring finger can’t play correct Es (E4/E5/E6) without the opening of the RH little finger hole (which controls D/D#) in our design. Henry said if the RH ring fingerhole wants to be a real major hole and keeps a healthy octave relationship, one of the possibilities is to have a much bigger hole and place it 5 mm lower. The overall tuning result looks acceptable, but whether this further stretch that leads to a spacing of 39–40mm could be manageable for flute players remains a question. At this time, we aim to set this spacing at a more comfortable range, but WIDesigner does provide an option if a player really requests a true major hole for the RH ring finger despite the expense of a more stretchy spacing. Therefore, a better description of our current flute might be: it has 5 major, 3 extra, and 2 equally important holes.

For the 3 extra holes, I think your advice is great. I’m not quite sure what a minor flute means. For our flute, if it is a 6-hole flute, a simple scale it can play is the D major scale. Probably you mean making F natural instead of F# tonehole as part of the 6-hole. It then creates a d melody minor scale with ease.

We indeed thought of setting a closed hole as the membrane hole, but later we found out the impact of the membrane on tuning is more than that. Chen-Gia Tsai in his dissertation “The Chinese membrane flute (dizi): physics and perception of its tones” provides the following figure describing the effect of the membrane to each note of a Chinese flute in G (as its lowest note) measuring the input impedance of each.

<img width="852" alt="截圖 2024-06-11 上午11 13 40" src="https://github.com/edwardkort/WWIDesigner/assets/85783979/86087e5c-48f7-4550-9521-42c1fc6774cf">

For the first two octaves, two tendencies can be observed. For notes G4/5, A4/5, and B4/5 the membrane mainly made them flat. For notes C, D, E, and F, the membrane stretch their octave relationships. For notes of the third octave, the membrane flattens them very much. These tendencies are quite nonlinear but are predictable once a flute has its membrane hole position in similar proportions with this flute in G. As far as we know, a closed hole or a gap created by unequal lengths of sockets and tenons impacts linearly on tuning. While it might be possible to simulate the membrane’s nonlinear effects using multiple linear models (guessed by Henry), he chose to directly counteract possible known tendencies created by the membrane. 

Tsai gives a full description on reasons why the dizi membrane behaves in this way. In my simple understanding, it relates to different positions of pressure nodes and antinodes of standing waves of different pitches of the flute relative to a fixed position of the membrane. Using Tsai’s example of the dizi in G, E5 produces a pressure antinode at the membrane position while E6 produces a pressure node. This makes the resonance of E5 heavily impacted by the membrane and that of E6 not affected. It implies that the position of the membrane is important and Tsai seems to discuss only one position of the membrane which is approximately in the middle between the embouchure hole and the first nearest tonehole. With some experiments and thoughts, we think the membrane will cause more average impact on pitches if it is placed nearer the embouchure hole, locating on more notes’ pressure antinodes. However, its average brightness will not sound like a Chinese dizi which people regard should own both bright and dark timbres in one instrument. Maybe some aesthetics are functioning.
